src/components/Images/ImageDetails.jsx

```diff
@@ -54,7 +54,7 @@ function ImageDetails() {
             force: forceDelete,
             noPrune: noPrune
         }).then((res) => {
-            toast.success(res);
+            toast.success('Image successfully deleted!');
```

Comment:
Understood, react-toastify swallowed it without rendering a toast because it was an object, rather than a string, whereas sweet alert will try to render it resulting in the classic [object object] type thing.